gtk3/sample/misc/scalebutton.rb

```diff
@@ -2,18 +2,30 @@
 =begin
   scalebutton.rb - Very simple Gtk::ScaleButton example.
 
-  Copyright (c) 2007 Ruby-GNOME2 Project Team 
+  Copyright (c) 2007-2015 Ruby-GNOME2 Project Team
   This program is licenced under the same licence as Ruby-GNOME2.
-
-  $Id: scalebutton.rb,v 1.1 2007/07/12 14:04:54 ggc Exp $
 =end
 
-require 'gtk3'
+require "gtk3"
 
 window = Gtk::Window.new
-window.add(scale = Gtk::ScaleButton.new(Gtk::IconSize::IconSize::BUTTON))
-scale.set_icons(['gtk-goto-bottom', 'gtk-goto-top', 'gtk-execute'])
-scale.signal_connect('value-changed') { |widget, value| puts "value changed: #{value}" }
+window.set_size_request(300, 300)
+scale = Gtk::ScaleButton.new(Gtk::IconSize::BUTTON, 0, 100, 2, nil)
```

Comment:
It seems that all arguments should be omitable. And they should be able to be specified by `Hash` such as `:icon_size => :button`.

I'll do it tomorrow. If you can do it before I do it, could you create a pull request?